sentry/values.yaml

```diff
@@ -245,6 +215,81 @@ filestore:
 
 config:
   yaml: |
-    # YAML
```

Comment:
Similar question. Old chart calls this `configYml` and `sentryConfPy`

Reply:
Thanks for your report. I'll take care of that in the next PR

Reply:
The good thing is upgrade paths deal with this. It's less of an issue as config maps are different anyway. So it's not a huge problem for people to change the values.